### ISSUE TITLE
Accessibility Issue:  Cannot Access Menu Items Using Keyboard

### ISSUE THREAD

**Issue by VanitaW** [topcoder-platform/community-app#2504]
***Reproduction Steps:***
1. Go to https://test-community-app.topcoder-dev.com/challenges 
2. Sign in with username: dan_developer  password: dantopcoder123
3. Click on 'log in'
4.  Navigate the site using only the keyboard.
5. Observe that  the navigation bar is inaccessible using the keyboard

***Expected Behavior***
To be able to navigate the menu using only the keyboard

***Actual Behavior***
Nothing happens when the keyboard is pressed

***Device/OS Information:***
macOS
Safari
12.1.1

**Comment by LieutenantRoger** [comment]
Please take a look and see if this is the expected: https://github.com/topcoder-platform/navigation-component/pull/122
